siteimprove/admin/class-siteimprove-admin.php

```diff
@@ -170,7 +170,7 @@ private function siteimprove_add_js( $url, $type ) {
 		if ( ! empty( $file_name ) && preg_match( $pattern, $file_name ) ) {
 			$overlay_path = Siteimprove::JS_LIBRARY_URL . $file_name;
 		} else {
-			if ( $newest_version ) {
+			if ( $disabled_new_version ) {
```

Comment:
Shouldn't this be the other way around? I'd expect "disable new version" would get v1...

Reply:
[For Flavio to continue on, added new text from product](https://github.com/Siteimprove/CMS-plugin-Wordpress/pull/30/commits/0a6ac231a5925c795bc3efed96b745dc71d01fa6)

We are implementing the new default checked on. Perhaps we should put it back into draft PR, while that's being implemented.